test/functional/specs/Identity/C5287654.js

```diff
@@ -22,15 +22,14 @@ import { MAIN_IDENTITY_COOKIE_NAME } from "../../helpers/constants/cookies";
 import createAlloyProxy from "../../helpers/createAlloyProxy";
 import createNetworkLogger from "../../helpers/networkLogger";
 import getResponseBody from "../../helpers/networkLogger/getResponseBody";
-import RefererHook from "../../helpers/requestHooks/refererHook";
```

Comment:
Apparently, the testCafe framework didn't like how we were setting the referrer before.